main.tf

```diff
@@ -1,7 +1,7 @@
 locals {
   enabled               = module.this.enabled
   partition             = join("", data.aws_partition.current[*].partition)
-  directory_bucket_name = var.create_s3_directory_bucket ? "${local.bucket_name}-${var.availability_zone_id}" : ""
+  directory_bucket_name = var.s3_directory_bucket_enabled ? "${local.bucket_name}-${var.availability_zone_id}" : ""
```

Comment:
This is a breaking change...

The cloudposse naming convention is to use `_enabled`

The `create_` convention is from terraform-aws-modules